modules/navier_stokes/test/tests/finite_volume/ins/channel-flow/2d-mixing-length-physics.i

```diff
@@ -0,0 +1,89 @@
+Re = 1e4
+von_karman_const = 0.2
+
+D = 1
+rho = 1
+bulk_u = 1
+mu = '${fparse rho * bulk_u * D / Re}'
+
+advected_interp_method = 'upwind'
+
+[Mesh]
+  [gen]
+    type = GeneratedMeshGenerator
+    dim = 2
+    xmin = 0
+    xmax = 5
+    ymin = 0
+    ymax = '${fparse 0.5 * D}'
+    nx = 20
+    ny = 10
+    bias_y = '${fparse 1 / 1.2}'
+  []
+[]
+
+[Physics]
+  [NavierStokes]
+    [WCNSFVFlow]
+      [flow]
+        compressibility = 'incompressible'
+
+        density = ${rho}
+        dynamic_viscosity = ${mu}
+
+        initial_velocity = '1 1 0'
+        initial_pressure = 0.0
+
+        inlet_boundaries = 'left'
+        momentum_inlet_types = 'fixed-velocity'
+        momentum_inlet_function = '1 0'
+
+        wall_boundaries = 'top bottom'
+        momentum_wall_types = 'noslip symmetry'
+
+        outlet_boundaries = 'right'
+        momentum_outlet_types = 'fixed-pressure'
+        pressure_function = '0'
+
+        momentum_advection_interpolation = ${advected_interp_method}
+        mass_advection_interpolation = ${advected_interp_method}
+      []
+    []
+    [WCNSFVScalarAdvection]
+      [scalars]
+        add_scalar_equation = true
```

Comment:
It seems a little redundant to have `add_scalar_equation` two lines after the `WCNSFVScalarAdvection` label. Is there anything we can do about that?

Reply:
looking at your other test inputs, it seems like this parameter is not necessary which is great

Reply:
I ll do a sweep to remove those unnecessary ones. I started out with something more verbose than now

Reply:
done